core/src/main/kotlin/Formats/StructuredFormatService.kt

```diff
@@ -55,6 +56,29 @@ abstract class StructuredFormatService(locationService: LocationService,
         return StringBuilder().apply { formatText(location, content, this, listKind) }.toString()
     }
 
+    fun ContentNode.ignoreParagraphAtTop(): ContentNode {
```

Comment:
This logic is already handled by the buildInlineContentTo() function; doesn't need to be duplicated.